### ISSUE TITLE
Icon Request: fa-fiverr

### ISSUE THREAD

**Issue by matrixdevuk** [FortAwesome/Font-Awesome#5970]
I'd love a Fiverr icon (fiverr.com).

Could you add this when you get some time? Thanks. :)

**Comment by emmaleonil** [comment]
+1

**Comment by MysteryDash** [comment]
+1

**Comment by 9bytes** [comment]
has this been implemented yet? I can't seem to get it in my footer...

**Comment by MysteryDash** [comment]
Nope it hasn't been implemented.

**Comment by 9bytes** [comment]
-.- do you know if fiverr has brand assets that would be ok for these?

**Comment by MysteryDash** [comment]
no, sorry

2015-11-08 18:32 GMT+01:00 9bytes <notifications@github.com>:

**Comment by 9bytes** [comment]
ok thanks...

**Comment by tuzzmaniandevil** [comment]
:+1:

**Comment by arlensot** [comment]
👍

**Comment by smallminddesign** [comment]
:+1:

**Comment by ibnuh** [comment]
👍

**Comment by FaizanDurrani** [comment]
Bump?

**Comment by Joseph-NC** [comment]
Just hope fiverr will be added soon.
But this looks like fiverr - fa-fonticons

**Comment by brettsloan** [comment]
+1
Anyone know if this is being on worked on or a substitute I could use in the mean time?

**Comment by Selah1337** [comment]
Yes, a fiverr icon similar to this would be great.
![untitled-1](https://user-images.githubusercontent.com/31610416/40330131-f8fd2880-5d19-11e8-8e7b-a326fa9d48eb.png)

**Comment by Legittti** [comment]
+1

**Comment by Harrison-B** [comment]
+1

**Comment by alexfares** [comment]
+1

**Comment by MarkVDD** [comment]
+1

**Comment by OliverCordingl1** [comment]
+1

**Comment by tagliala** [comment]
This request doesn't show up in search 🤔

Tried to force a title change

**Comment by MaheshNat** [comment]
Please make a fiverr icon, would benefit freelancers who want icons for these on their portfolios (like me). Upwork, freelancer.com, people per hour, and others would also help

**Comment by OliverCordingl1** [comment]
Why is this being ignored? This icon would benefit so many people. Is it possibly because of copyright/trademark infringement? Surely it would be good for Fiverr if their logo was used in FontAwesome as it could lead more people to their site.

**Comment by MaheshNat** [comment]
I completely agree, beneficial for both fiverr and those who use the logo on their sites.

**Comment by tagliala** [comment]
> Why is this being ignored?

Hi, this request is not being ignored :)

> This icon would benefit so many people.

I understand the value that this icon may have for some use cases. Please take a look at our [Brand Icon Leaderboard](https://fontawesome.com/community/leaderboard/brands) and [Icon Leaderboard FAQs](https://fontawesome.com/community/leaderboard/new#faqs), they contain a lot of information about how requests work

If you have a strong community of developers interested in Fiverr, please ask them to upvote this request (by using 👍 , as per Contributing Guidelines)

**Comment by lydocia** [comment]
Sad to see this still hasn't been added. :(

**Comment by fxmonk** [comment]
> I'd love a Fiverr icon (fiverr.com).
> 
> Could you add this when you get some time? Thanks. :)

Needed

**Comment by Bishwas-py** [comment]
We also need Fiverr icon in Font Awesome. +1

**Comment by Bishwas-py** [comment]
Till now, I have been using `fab fa-fonticons-fi`

**Comment by SaabK** [comment]
i really need it

**Comment by pssolanki111** [comment]
+1

**Comment by raymukh** [comment]
+1

**Comment by mohsinamerriaz** [comment]
+1

**Comment by mrwebwork** [comment]
+1

**Comment by utenor** [comment]
+1

**Comment by sunwooz** [comment]
👍

**Comment by juush** [comment]
+1

**Comment by xucian** [comment]
still not available?

**Comment by Bishwas-py** [comment]
> still not available?

go with iconify boss, they have a huge list of icons

**Comment by cjoncmelero** [comment]
+1

https://icon-sets.iconify.design/simple-icons/fiverr/

**Comment by umairhanifdev** [comment]
when fa-fiverr will be added? really needed for portfolio/personal website. :-(

**Comment by IYSTREEM** [comment]
1+

**Comment by mrwebwork** [comment]
> > Why is this being ignored?
> 
> Hi, this request is not being ignored :)
> 
> > This icon would benefit so many people.
> 
> I understand the value that this icon may have for some use cases. Please take a look at our [Brand Icon Leaderboard](https://fontawesome.com/community/leaderboard/brands) and [Icon Leaderboard FAQs](https://fontawesome.com/community/leaderboard/new#faqs), they contain a lot of information about how requests work
> 
> If you have a strong community of developers interested in Fiverr, please ask them to upvote this request (by using 👍 , as per Contributing Guidelines)

I noticed that the Fiverr icon is currently ranked within the top 10 on the [brand icons leaderboard](https://fontawesome.com/leaderboard/brands) and has been a highly anticipated addition for nearly a decade, as seen in [Issue #5970](https://github.com/FortAwesome/Font-Awesome/issues/5970).

Given the longstanding interest and high demand for the Fiverr icon, could you provide an update on when it might be officially added to Font Awesome? I would greatly appreciate any information on its status.

**Comment by iayushanand** [comment]
how long is it gonna take to be added its been over 4 years and it's still not added

**Comment by jaslam94** [comment]
+1

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.
